src/main/java/nextstep/courses/domain/SessionRepository.java

```diff
@@ -0,0 +1,16 @@
+package nextstep.courses.domain;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface SessionRepository {
+    int save(Session session);
+
+    Session findById(Long id);
+    List<Session> findByCourseId(Long id);
+
+    int update(Session session);
+
+    int delete(Long id);
+    Optional<List<Session>> findBySessionIds(List<Long> sessionIds);
```

Comment:
```suggestion
   List<Session> findBySessionIds(List<Long> sessionIds);
```
값이 없는 경우 빈 List 콜렉션을 반환하면 되기 때문에 콜렉션의 경우 굳이 Optional로 구현할 필요가 있을까?